src/app/me/tickets/ticket.component.tsx

```diff
@@ -87,125 +88,129 @@ export function Ticket({ ticket, refetch, selected, onSelect }: TicketProps) {
 
   return (
     <>
-      <div className="group relative mb-2 flex h-28 space-x-4 rounded-lg border p-2 shadow transition hover:shadow-lg">
-        <div
-          className={`${!selected && "invisible"} absolute left-2 top-2 ${ticket.state === "open" ? "group-hover:visible" : ""}`}>
-          <Checkbox checked={selected} onClick={() => onSelect(!selected)} />
-        </div>
-        <div className="hidden w-32 items-center justify-center self-stretch md:flex">
-          <Image
-            alt={"Tonik"}
-            src={"/images/tonik.svg"}
-            width={120}
-            height={120}
-          />
-        </div>
-        <div className={"flex flex-1 flex-col justify-end"}>
-          <div className={"overflow-hidden whitespace-nowrap"}>
-            <span className={"font-bold"}>{conference?.name}</span>
+      <Card className="mb-4">
+        <CardContent className="group relative flex h-28 space-x-4 p-4">
+          <div
+            className={`${!selected && "invisible"} absolute left-2 top-2 ${ticket.state === "open" ? "group-hover:visible" : ""}`}>
+            <Checkbox checked={selected} onClick={() => onSelect(!selected)} />
           </div>
-          <div className={"text-sm text-gray-500"}>
-            {conference?.ticketDescription}
+          <div className="hidden w-32 items-center justify-center self-stretch md:flex">
+            <Image
+              className={"bg-current text-primary"}
+              alt={"Tonik"}
+              src={"/images/tonik.svg"}
+              width={120}
+              height={120}
+            />
           </div>
-          <div className={"flex flex-1 items-end justify-end"}>
-            {!ticket.transferId && ticket.state === "open" && (
-              <>
-                <ConfirmDialog
-                  title={"Do you want to transfer this ticket?"}
-                  description={
-                    "This will create a code on the ticket that can be used to redeem the ticket. While the ticket is in transfer it cannot be used to check in"
-                  }
-                  onConfirm={async () => {
-                    await createTicketTransfer()
-                    await refetch()
-                  }}>
+          <div className={"flex flex-1 flex-col justify-end"}>
+            <div className={"overflow-hidden whitespace-nowrap"}>
+              <span className={"font-bold"}>{conference?.name}</span>
+            </div>
+            <div className={"text-sm text-gray-500"}>
```

Comment:
this should probably be changed from `text-gray-500` to `text-muted`, no? :)

Reply:
text-muted made it unreadable in dark mode.

Reply:
That's why text-muted should automatically be adjusted for dark mode and light mode, via the css variables 😄